
The usual sloppy C typing for gcc >= 3.4.x.

  - Rene Rebe <rene@exactcode.de>

--- ./plug-ins/psd/psd.c.vanilla	2004-09-24 09:52:38.577481624 +0200
+++ ./plug-ins/psd/psd.c	2004-09-24 09:53:21.555653865 +0200
@@ -463,7 +463,7 @@
 }
 
 
-static GDrawableType
+static GImageType
 psd_type_to_gimp_type (psd_imagetype psdtype)
 {
   switch(psdtype)
